### ISSUE TITLE
Contact entries in Saros view are not sorted correctly

### ISSUE THREAD

**Issue by tobous** [saros-project/saros#610]
Currently, the contacts listed in the Saros view retain the order they were returned by the roster. Furthermore, this order does not always seem to be consistent between lifecycles (and also across the lifecycle of the plugin). When another participant leaves a session, their account is not correctly added to the list of contacts. Even if they are still online, they are added to the end of the list.

It would be preferable to sort the accounts alphabetically to provide a consistent ordering for the user. This might not be trivial as the accounts are held as a tree, which would have to be sorted every time it is modified.

#### Expected Behavior
Accounts are sorted into two groups:
- conntected
- not conntected

Each group is sorted alphabetically. The "connected" group is displayed first, the "not connected" group second. A separator between the groups is acceptable but not necessary in my opinion.

**Comment by srossbach** [comment]
This is not that hard.

Ordering used in Eclipse https://github.com/saros-project/saros/blob/master/eclipse/src/saros/ui/model/roster/RosterComparator.java

When we receive updates we are throwing all data away (you can do that in Swing too)

https://github.com/saros-project/saros/blob/master/eclipse/src/saros/ui/model/roster/RosterContentProvider.java#L49

https://github.com/saros-project/saros/blob/master/intellij/src/saros/intellij/ui/tree/ContactTreeRootNode.java#L164

So basically you fetch all roster entries, sort them and then add them.
